pylib/cqlshlib/test/test_cqlsh_output.py

```diff
@@ -701,21 +701,24 @@ def test_describe_columnfamily_output(self):
                 tinyintcol tinyint,
                 uuidcol uuid,
                 varcharcol text,
-                varintcol varint
+                varintcol varint,
+                PRIMARY KEY (num)
```

Comment:
Seeing these changes made me convinced that the way this test is written should be changed. It shouldn't compare the full string, but rather the content. I mean, what if in the future we decide to sort the column names differently (I think there's a discussion about that right now? To format the WITH below differently, and so on? It will break this test.

By the way, won't this test break every time we add a new WITH parameter to Scylla? I already see you added "cdc" and "paxos_grace_seconds", we'll need to add more and more of those :-( Are we fine with that?

Oh, and don't we allow someone to use cqlsh with Cassandra? It seems this test won't work with Cassandra.

I don't know what to recommend instead of this method. I can think of two ideas:
1. Use "describe" to get the description of a table and then create this table again and "describe" it again and check the two descriptions are identical.
2. Compare the expected strings after passing them through some sort of "canonization" function, which removes whitespace, sorts columns and WITH options, canonizes floating point values, etc. This won't be easy. I don't know what to recommend about non-standard options like cdc and paxos_grace_seconds.

Reply:
> By the way, won't this test break every time we add a new WITH parameter to Scylla? 

Probably, but I am not sure if refactoring these tests should be a part of this PR, maybe we should create a separate issue to change it?

> Oh, and don't we allow someone to use cqlsh with Cassandra? It seems this test won't work with Cassandra.

In `test_describe_columnfamily_output` there are two different versions of expected output, we use different one if scylla is used and I only changed that part so it should work fine with Cassandra.

Reply:
now the test are a bit more easy to grok, and it's o.k. from my POV